pkg/symbolizer/symtab_linux.go

```diff
@@ -123,13 +146,13 @@ func (s *Symbolizer) newSymbolTableFromPid(pid uint32, expectedExeKey exeKey) (*
 		return nil, errors.New("getting syscall.Stat_t failed")
 	}
 	ino := stat.Ino
```

Comment:
nit: this variable can be removed and use stat.Ino directly on the error message.